### ISSUE TITLE
NameError: global name 'ResolutionError' is not defined

### ISSUE THREAD

**Issue by bobrik** [projectcalico/felix#1168]
```
2016-11-08 01:08:24,232 [INFO][186/2] calico.felix.fetcd 594: etcd driver status changed to in-sync
2016-11-08 01:08:24,236 [ERROR][186/27] calico.felix.futils 502: Exception in wrapped function <function _periodically_usage_report at 0x7fc6b46455f0>
Traceback (most recent call last):
  File "site-packages/calico/felix/futils.py", line 500, in wrapped
  File "site-packages/calico/felix/fetcd.py", line 363, in _periodically_usage_report
NameError: global name 'ResolutionError' is not defined
Traceback (most recent call last):
  File "site-packages/gevent/greenlet.py", line 534, in run
  File "site-packages/calico/felix/futils.py", line 500, in wrapped
  File "site-packages/calico/felix/fetcd.py", line 363, in _periodically_usage_report
NameError: global name 'ResolutionError' is not defined
<Greenlet at 0x7fc6b3cde4b0: <bound method _FelixEtcdWatcher._periodically_usage_report of <_FelixEtcdWatcher at 0x7fc6b3cde410>>> failed with NameError
```

This is on `calico/node:v0.23.0`.

**Comment by fasaxc** [comment]
Hmm, looks like someone accidentally deleted the import for that exception?

**Comment by fasaxc** [comment]
No, it was never there; but the name of the exception only gets resolved if an exception is raised in that block.  If the `try:` succeeds, then the `NameError` isn't raised:
```
>>> try:
...     pass
... except Foo:
...     pass
... 
>>> try:
...     raise Exception()
... except Foo:
...     pass
... 
Traceback (most recent call last):
  File "<input>", line 3, in <module>
NameError: name 'Foo' is not defined
```

**Comment by fasaxc** [comment]
I have a WiP fix for the NameError but the error doesn't identify the root cause.  I'll add a log into the patch to trace out the root cause.

**Comment by fasaxc** [comment]
Fixed under #1183
